packages/api/src/models/medical/cq-directory.ts

```diff
@@ -0,0 +1,59 @@
+import { DataTypes, Sequelize } from "sequelize";
+import { CQOrganization } from "../../domain/medical/cq-directory";
+import { BaseModel, ModelSetup } from "../../models/_default";
+import { Organization } from "@metriport/carequality-sdk/models/organization";
+
+export class CQDirectoryModel extends BaseModel<CQDirectoryModel> implements CQOrganization {
+  static NAME = "cq_directory";
+  declare oid: string;
+  declare name?: string;
+  declare urlXCPD: string;
+  declare urlDQ?: string;
+  declare urlDR?: string;
+  declare latitude?: string;
+  declare longitude?: string;
+  declare data?: Organization;
+  declare state?: string;
+
+  static setup: ModelSetup = (sequelize: Sequelize) => {
+    CQDirectoryModel.init(
+      {
+        ...BaseModel.attributes(),
+        oid: {
+          type: DataTypes.STRING,
+        },
+        name: {
+          type: DataTypes.STRING,
+        },
+        urlXCPD: {
+          type: DataTypes.STRING,
+          field: "url_xcpd",
```

Comment:
AFAIK we only need to map from camelCase to snake_case on the migration, no?

Reply:
ill double check, but i think it was failing for me until i added it, saying "url_x_c_p_d" is not a valid column or whatever

Reply:
yea, just confirmed that this doesn't work.. 
`sql: `SELECT "id", "created_at" AS "createdAt", "updated_at" AS "updatedAt", "version", "oid", "name", "url_x_c_p_d" AS "urlXCPD"...`

I see we use camelCase in other models, but maybe since this is not exactly camelCase, and it has a bunch of capital letters in a row, it's treated differently?

Reply:
`hint: 'Perhaps you meant to reference the column "CQOrganizationModel.url_xcpd".',`

Reply:
yea, Ive seen that example before, which is why I'm a little confused as well... but really, if I take it out, it starts erroring, and I confirmed it today.. 🤷‍♂️

Reply:
I'll keep it as is if that's ok